app/controllers/resources_controller.rb

```diff
@@ -16,7 +16,7 @@ def show
 
   def search
     result = Resources::Search.perform(params.require(:query), sort_order, scope: resources)
-    render json: ResourcesPresenter.present(result)
+    render json: {resources: ResourcesPresenter.present(result)}
```

Comment:
I'm not sure why I had to nest this explicitly, shouldn't the presenter take care of this? Maybe because the return from the search is an Array of resources and not an ActiveRecord Association?